aimmo_runner/setup.cfg

```diff
@@ -0,0 +1,13 @@
+[bdist_wheel]
+universal = 1
+
+[pep8]
+max-line-length = 160
+
+[versioneer]
+VCS = git
+style = pep440-pre
+versionfile_source = ../players/_version.py
```

Comment:
hmm could that be set up elsewhere so that it's less susceptible to relative path changes?

Reply:
We have a root directory dependant on where the project is ran. If it's local, then it's your typical path, but Travis is different, so this is checked at the start and saved in the `runner.run` file. We could use that variable here, but it would take a lot of work to get it passed from python to a `.cfg` file. If you think it's important I could start looking into it?

Reply:
More things to maintain though ;( I'll see if we can maybe test this, or if it's already done

Reply:
Could you write an issue for this?

Reply:
#435